PBuild/Recipe.pm

```diff
@@ -82,7 +82,7 @@ sub find_recipe {
     }
   }
   if (1) {
-    @files = grep {/mkosi\.$/} keys %files;
+    @files = grep {/^mkosi\./} keys %files;
```

Comment:
I probably never noticed because I set `Type: mkosi` in prjconf, so this commit looks good to me